tests/test-sources/plugins/by-name/treesitter-textobjects/default.nix

```
@@ -12,71 +12,73 @@
       treesitter-textobjects = {
         enable = true;
 
-        select = {
-          enable = true;
-          disable = [ ];
-          lookahead = true;
-          keymaps = {
-            af = "@function.outer";
-            "if" = "@function.inner";
-            ac = "@class.outer";
-            ic = {
-              query = "@class.inner";
-              desc = "Select inner part of a class region";
+        settings = {
+          select = {
+            enable = true;
+            disable = [ ];
```

Shouldn't we be using actual empty tables? As i understand the original is equal to not having this line.
```suggestion
            disable.__empty = { };
```

---

> this error is common across the entire test suite.

I worded it as a question because it is so common that I wasn't sure if I was correct.

> Maybe we should go and do a tree-wide change to use __empty and __raw "nil".

This seems pretty fun to me. I'll see if i can modify my WIP Nix linter to do this automatically.